pkg/probeflags/probeflags.go

```diff
@@ -0,0 +1,21 @@
+package probeflags
+
+import "fmt"
+
+// Probe arguments -- can be Flagified
+const (
+	ArgUrl            = "url"
```

Comment:
GoIsm™: acronyms should be all uppercase:
```suggestion
	ArgURL            = "url"
```

Reply:
I'm still seeing it as ArgUrl 🤔